### DISCUSSION TITLE
iOS app?

### DISCUSSION THREAD

**Discussion by zssfred** [community/community#3418] [Mobile] [answered]
iOS app?

**Comment by PranamBhat** [comment, marked as answer]
Hi @zssfred 

At the **GitHub** Universe event in San Francisco, the company announced that it will release official Android and iOS apps that will enable developers to reply to questions from users, read bug reports, and assign issues to other developers from their phones.

The app first launched in beta on **iOS** in **November 2019** and on **Android** in **January 2020**.


Download here : 

[https://github.com/mobile](https://github.com/mobile)

[https://apps.apple.com/us/app/github/id1477376905?ls=1](https://apps.apple.com/us/app/github/id1477376905?ls=1)

**Comment by zssfred** [comment]
Thank you so much!

Fred

**Reply by PranamBhat** [reply]
@zssfred  You're welcome.

Kindly mark this as "**Answered**" and leave the discussion.

Thank you!
